docs/en/sql-reference/functions/string-functions.md

```diff
@@ -649,3 +649,54 @@ Result:
 -   [List of XML and HTML character entity references](https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references)
 
 
+
+## extractTextFromHTML {#extracttextfromhtml}
+
+The function extracts text from HTML or XHTML according to the following rules.
+
+1. Comments starting with `<!--` and ending with `-->` are removed.
+1. The content of a `CDATA` section between `<![CDATA[` and `]]>` is left as is, without furthure processing. Note that it is appended to the previous text without any space.
+1. A text wrapped with `<script>` or `<style>` tags is removed entirely. If `script` or `style` are the names of XML namespaces (like `<script:a>`) then they are treated like usual tags.
+1. Any tag is replaced with a space. Note that elements like `<>`, `<!>`, `<!-->` are also replaced. Tag without closing bracket `>` is removed to the end of an input text. 
+1. Any sequence of whitespaces (space, new line, carriage return, tab, vertical tab or form feed) is converted to a single space.
```

Comment:
```suggestion
1. Any sequence of whitespaces (space, new line, carriage return, tab, vertical tab or form feed) is converted to a single space. But if text is separated by other elements and there is no whitespace, it is inserted.
```

Reply:
The text must be verbatim copy-paste of the comment in the source code.
No edits acceptable.

Reply:
Description very close to the comment in the source code is ready.
@alexey-milovidov, have a look, please.